spec/fixtures/companies.yml

```diff
@@ -0,0 +1,49 @@
+# == Schema Information
+#
+# Table name: companies
+#
+#  id                    :integer          not null, primary key
+#  name                  :string
+#  web                   :string
+#  email                 :string
+#  phone                 :string
+#  partnermanager        :string
+#  contact_person        :string
+#  email_contact_person  :string
+#  phone_contact_person  :string
+#  crm                   :string
+#  level                 :string
+#  my_company            :boolean, default: false
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#
+
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+firma:
+  name: Firma
+  web: www.example.com
+  email: info@example.com
+  phone: +41 00 000 00 00
+  partnermanager: Urs Müller
+  contact_person: Philip Müller
+  email_contact_person: pmueller@example.com
+  crm: crm
+  level: A
+  my_company: true
+  updated_at: 2016-10-20 13:15:11
+  created_at: 2016-10-20 13:15:11
```

Comment:
normally you don't have to set update/created attrs in fixtures.
i just checked and we do that in other fixtures too.
i would suggest to just remove this attributes ... if not required for some reasons